### ISSUE TITLE
GitHub Actions `kayak/ecspresso@v1` failed due to v1 missing in the released list URL

### ISSUE THREAD

**Issue by seiichi1101** [kayac/ecspresso#559]
I got errors like the below log.

According to the [reference](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases), the list release URL `https://api.github.com/repos/kayac/ecspresso/releases` returns the latest 30 releases as default.
However, v1 releases are not included in 30 outs, so pagination handling is required to fix it.

- Workflow settings

```
...
      - uses: kayac/ecspresso@v1

      - name: ECS deploy
        run: ecspresso deploy --envfile=ecs/prd.env
...
```

- Error Logs

```
Run kayac/ecspresso@v1

Run set -e
  set -e
  VERSION="latest"
  if [ "${VERSION}" = "latest" ]; then
    DOWNLOAD_URL=$(curl -sS https://api.github.com/repos/kayac/ecspresso/releases | jq -r '[.[]|select(.tag_name < "v1.99")|select(.prerelease == false)][0].assets[].browser_download_url|select(match("linux.amd64."))')
  else
    DOWNLOAD_URL=https://github.com/kayac/ecspresso/releases/download/${VERSION}/ecspresso_${VERSION:1}_linux_amd64.tar.gz
  fi
  cd /tmp
  curl -sfLO ${DOWNLOAD_URL}
  if [[ "${DOWNLOAD_URL}" =~ \.tar\.gz$ ]]; then
    FILENAME=$(basename $DOWNLOAD_URL .tar.gz)
    tar xzvf ${FILENAME}.tar.gz
    sudo install ecspresso /usr/local/bin/ecspresso
  elif [[ "${DOWNLOAD_URL}" =~ \.zip$ ]]; then
    FILENAME=$(basename $DOWNLOAD_URL .zip)
    unzip ${FILENAME}.zip
    sudo install ${FILENAME} /usr/local/bin/ecspresso
  fi
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}

...
jq: error (at <stdin>:8371): Cannot iterate over null (null)
Error: Process completed with exit code 5.
```

**Comment by fujiwara** [comment]
@seiichi1101 Thank you for reporting!

At first, ecspresso v1 will not be released in the future. So I recommend using `kayac/ecspresso@v2` with `version` specified. @v2 can install any released version, including v1.x.

Typically, I would recommend specifying the version number on the production environment. The latest version has problems in rare cases.

But I'll fix this problem.

**Comment by fujiwara** [comment]
I fixed this by [3a8cbcf](https://github.com/kayac/ecspresso/commit/3a8cbcfc8577e5ca4dd428a622bb13e9ba83d57b), but it is a mitigation.

**Comment by fujiwara** [comment]
https://github.com/kayac/ecspresso/blob/8c8c8b2da19a786d9f1e73e6f4f4894e362e782d/action.yml#L13-L17

I've statically set the `latest` version of the v1 branch to `v1.7.15` and added the deprecation message.
